ext/standaloneusers/CRM/Standaloneusers/Page/ChangePassword.php

```diff
@@ -0,0 +1,21 @@
+<?php
+use CRM_Standaloneusers_ExtensionUtil as E;
+
+class CRM_Standaloneusers_Page_ChangePassword extends CRM_Core_Page {
+
+  public function run() {
+    // Example: Set the page-title dynamically; alternatively, declare a static title in xml/Menu/*.xml
+    CRM_Utils_System::setTitle(E::ts('Change Password'));
```

Comment:
Per the boilerplate comment, the title is indeed static and set in the xml, so it looks like this boilerplate can be deleted.